src/main/kotlin/uk/gov/justice/digital/assessments/services/QuestionService.kt

```diff
@@ -158,22 +161,13 @@ class QuestionService(
   }
 
```

Comment:
@SWhitlam i have removed the coupling from then frontend endpoint to get groups by code and also uuid, this was necessary for the recursive load of groups so I have refactor that slightly so we just encapsulate getting the groups by uuid when needed in the service layer. I would like to merge this PR to test all of this is working as expected in the FE before i remove the top layer from the grouping table and we migrate the loading of the assessments to the new assessment schema layer.